### PR TITLE
[13.0] [MIG] delivery_multi_destination: Migration to v13.0

### PR DESCRIPTION
**Pull request by CarlosRoca13**
cc @Tecnativa TT26765

please @pedrobaeza @ernestotejeda review this

### CODE REVIEW

**Review by ernestotejeda — COMMENTED** [review]
Code review :+1:

File: delivery_multi_destination/tests/test_delivery_multi_destination.py [hunk 1]
```diff
@@ -0,0 +1,207 @@
...
+                    "default_order_id": order.id,
+                    "default_carrier_id": order.carrier_id.id,
+                }
+            )
+        )
+        choose_delivery_carrier = delivery_wizard.save()
+        choose_delivery_carrier.button_confirm()
+        sale_order_line = order.order_line.search(
+            [("name", "=", "Test carrier single")]
+        )
```
Comment (pedrobaeza) [inline delivery_multi_destination/tests/test_delivery_multi_destination.py]: You should search by `is_delivery` flag, and the search is not limited by the order, so it's searching globally:
```suggestion
        sale_order_line = order.order_line.filtered("is_delivery")
```
do the same on the rest

**Review by pedrobaeza — CHANGES_REQUESTED** [review]

**Comment by CarlosRoca13** [comment]
@pedrobaeza changes done

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**